static/js/compose.js

```diff
@@ -623,7 +584,6 @@ exports.respond_to_message = function (opts) {
     var msg_type;
     // Before initiating a reply to a message, if there's an
     // in-progress composition, snapshot it.
-    compose.snapshot_message();
```

Comment:
I think this should still snapshot the message, since it won't be hit by the focusout handler.